docs/developer/01-getting-started.md

```
@@ -0,0 +1,173 @@
+| ------------------ | -------- | ------------------------------------------------ |
+| `PUBLIC_LANDSCAPE` | `lapras` | Environment to use (lapras/pichu/pikachu/raichu) |
+
+## Common Issues
+
+### Issue: Nix flake not found
+
+**Symptom**: `error: experimental Nix feature 'nix-command' is disabled`
+
+**Solution**: Enable flakes in `~/.config/nix/nix.conf`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)